cmds/cpu/cpu9p_unix.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2022 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build !windows && !plan9
+// +build !windows,!plan9
+
+package main
+
+import (
+	"os"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hugelgupf/p9/p9"
+	"golang.org/x/sys/unix"
+)
+
+// SetAttr implements p9.File.SetAttr.
+func (l *cpu9p) SetAttr(mask p9.SetAttrMask, attr p9.SetAttr) error {
+	var err error
+	// Any or ALL can be set.
+	// We started out with this:
+	// err = multierror.Append
+	// for each case. It seemed clean. But it returns a non-nil
+	// error, even if all arguments are nil. Interesting.
+	// Don't believe HashiCorp's docs about result.ErrorOrNil():
+	// they're just flat-out wrong; even their example doesn't work.
+	// A setattr could include things to set,
+	// and a permission value that makes setting those
+	// things impossible. Therefore, do these
+	// permission-y things last:
+	// Permissions
+	// GID
+	// UID
+	// Since changing, e.g., UID or GID might make
+	// changing permissions impossible.
+	//
+	// The test actually caught this ...
+
+	if mask.Size {
+		if e := unix.Truncate(l.path, int64(attr.Size)); e != nil {
+			err = multierror.Append(e)
+		}
+	}
+	if mask.ATime || mask.MTime {
+		atime, mtime := time.Now(), time.Now()
+		if mask.ATimeNotSystemTime {
+			atime = time.Unix(int64(attr.ATimeSeconds), int64(attr.ATimeNanoSeconds))
+		}
+		if mask.MTimeNotSystemTime {
+			mtime = time.Unix(int64(attr.MTimeSeconds), int64(attr.MTimeNanoSeconds))
+		}
+		if e := os.Chtimes(l.path, atime, mtime); e != nil {
+			err = multierror.Append(e)
+		}
+	}
+
+	if mask.CTime {
+		// The Linux client sets CTime. I did not even know that was allowed.
+		// Change the create time of a file?
```

Comment:
ctime is "change time"

```

       Last status change timestamp (ctime)
              stat.st_ctime; statx.stx_ctime

              This is the file's last status change timestamp.  It is
              changed by writing or by setting inode information (i.e.,
              owner, group, link count, mode, etc.).
```

creation time is birth time (btime)

https://man7.org/linux/man-pages/man7/inode.7.html